<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TacheEditor</class>
 <widget class="QDialog" name="TacheEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label_fenetre">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Modification de la tâche </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_titre">
       <property name="text">
        <string>Titre</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_titre"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Date de disponibilité</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="editDispo"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Date d'échéance</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="editEcheance"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_duree">
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="text">
        <string>Durée</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="editDuree"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="checkBoxPreemp">
       <property name="text">
        <string>Preemptive</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonAnnuler">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSauvegarder">
       <property name="text">
        <string>Sauvegarder</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonAnnuler</sender>
   <signal>clicked()</signal>
   <receiver>TacheEditor</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
